ddl/ddl_api.go

```diff
@@ -638,7 +645,10 @@ func columnDefToCol(ctx sessionctx.Context, offset int, colDef *ast.ColumnDef, o
 	return col, constraints, nil
 }
 
-func getDefaultValue(ctx sessionctx.Context, col *table.Column, c *ast.ColumnOption) (interface{}, error) {
+// getDefault value will get the default value for column.
+// 1: get the expr restored string for the column which uses sequence next value as default value.
+// 2: get specific default value for the other column.
+func getDefaultValue(ctx sessionctx.Context, col *table.Column, c *ast.ColumnOption) (interface{}, bool, error) {
```

Comment:
Could we separate a function with signature `func isSequenceExpr(op *ColumnOption) bool`, instead adding another return value?

We had better adhere to the principle of 'single responsibility'.

Reply:
`getDefaultValue` will be consist of two func: `getSequenceDefault` & `getNormalDefault`.